nx v6.0.0-rc.2: Cannot find module 'strip-json-comments'

The following error occurs with `v6.0.0-rc.2` when creating a new workspace:
```
ng new "golocal" --collection=@nrwl/schematics
Cannot find module 'strip-json-comments'
Error: Cannot find module 'strip-json-comments'
    at Function.Module._resolveFilename (module.js:555:15)
    at Function.Module._load (module.js:482:25)
    at Module.require (module.js:604:17)
    at require (internal/module.js:11:18)
    at Object.<anonymous> (/private/var/folders/y_/z7673yw95yz1vg297jxvp_300000gn/T/tmp-6113RIuRwKJkU6Ke/node_modules/@nrwl/schematics/src/utils/ast-utils.js:9:25)
    at Module._compile (module.js:660:30)
    at Object.Module._extensions..js (module.js:671:10)
    at Module.load (module.js:573:32)
    at tryModuleLoad (module.js:513:12)
    at Function.Module._load (module.js:505:3)
child_process.js:614
    throw err;
    ^

Error: Command failed: /var/folders/y_/z7673yw95yz1vg297jxvp_300000gn/T/tmp-6113RIuRwKJkU6Ke/node_modules/.bin/ng new "golocal" --skip-install --collection=@nrwl/schematics
```

@vsavkin I can help look in PR but perhaps something quick?

--- a/packages/schematics/migrations/legacy-migrations/20180515-switch-to-nx6.ts
+++ b/packages/schematics/migrations/legacy-migrations/20180515-switch-to-nx6.ts
@@ -34,8 +34,13 @@ export default {
         execSync('npm install', { stdio: [0, 1, 2] });
       }
 
-      execSync('ng update @angular/cli@6.0.1', { stdio: [0, 1, 2] });
-      execSync('ng update @nrwl/schematics@6.0.0', { stdio: [0, 1, 2] });
+      execSync('ng update @angular/cli --from 1.7.4 --to 6.0.1', {
+        stdio: [0, 1, 2]
+      });
+      // TODO: remove --next before releasing 6.0.0 final
+      execSync('ng update @nrwl/schematics --from 1.0.3 --to 6.0.0 --next', {
+        stdio: [0, 1, 2]
+      });
     } catch (e) {
       console.warn(stripIndents`
         The automatic upgrade to Nx 6 has failed with the following error: ${e}.

--- a/packages/schematics/package.json
+++ b/packages/schematics/package.json
@@ -28,7 +28,7 @@
   },
   "homepage": "https://github.com/nrwl/nx#readme",
   "schematics": "./src/collection.json",
-  "ng-upgrade": {
+  "ng-update": {
     "requirements": {},
     "packageGroup": ["@nrwl/nx", "@nrwl/schematics"],
     "migrations": "./migrations/migrations.json"
@@ -45,6 +45,7 @@
     "npm-run-all": "4.1.2",
     "opn": "^5.3.0",
     "semver": "5.4.1",
+    "strip-json-comments": "2.0.1",
     "tmp": "0.0.33",
     "viz.js": "^1.8.1",
     "yargs-parser": "10.0.0",

